app/src/main/java/de/danoeh/antennapod/adapter/FeedItemlistAdapter.java

```diff
@@ -153,7 +141,9 @@ public View getView(final int position, View convertView, ViewGroup parent) {
                     holder.episodeProgress.setProgress(((ItemAccess) itemAccess).getItemDownloadProgressPercent(item));
                     holder.published.setVisibility(View.GONE);
                 } else {
-                    holder.episodeProgress.setVisibility(View.GONE);
+                    if(media.getPosition() == 0) {
```

Comment:
The only episode that should be showing that bar is one that's being played.  *UNLESS* you're specifically trying to show how far through a started episode someone is.